demo/firebase-idtoken-flow-example/src/App.tsx

```diff
@@ -0,0 +1,528 @@
+      const idTokenLoginParams = {
+        verifier: "web3auth-firebase-examples",
+        verifierId: parsedToken.sub,
+        idToken,
+      } as IdTokenLoginParams;
+
+      await coreKitInstance.loginWithJWT(idTokenLoginParams);
+
+      try {
+        let result = securityQuestion.getQuestion(coreKitInstance!);
+        if (result) setQuestion(result);
+      } catch {}
+
+      if (coreKitInstance.status === COREKIT_STATUS.REQUIRED_SHARE) {
+        uiConsole("required more shares, please enter your backup/ device factor key, or reset account"); 
+      } else {
+        completeSetup();
+      }
+      setCoreKitStatus(coreKitInstance.status)
+    } catch (error: unknown) {
+      uiConsole(error);
+    }
+  };
+
+  const completeSetup = async () => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance not found");
+    }
+    setProvider(coreKitInstance.provider);
+    const factorKeyMetadata = coreKitInstance.getCurrentFactorKey();
+    if (factorKeyMetadata.shareType === TssShareType.DEVICE) {
+      storeWebBrowserFactor(factorKeyMetadata.factorKey, coreKitInstance)
+    } else {
+      const factorKey = await coreKitInstance.createFactor({
+        shareType: TssShareType.DEVICE,
+      });
+      storeWebBrowserFactor(new BN(factorKey!, "hex"), coreKitInstance)
+    }
+  }
+
+  const getDeviceShare = async () => {
+    const factorKey = await getWebBrowserFactor(coreKitInstance!);
+    setBackupFactorKey(factorKey);
+    uiConsole("Device share: ", factorKey);
+  }
+
+  const inputBackupFactorKey = async () => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance not found");
+    }
+    if (!backupFactorKey) {
+      throw new Error("backupFactorKey not found");
+    }
+    const factorKey = new BN(backupFactorKey, "hex")
+    await coreKitInstance.inputFactorKey(factorKey);
+
+    if (coreKitInstance.status === COREKIT_STATUS.REQUIRED_SHARE) {
+      uiConsole("required more shares even after inputing backup factor key, please enter your backup/ device factor key, or reset account");
+    } else {
+      completeSetup();
+    }
+  }
+
+  const recoverSecurityQuestionFactor = async () => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance not found");
+    }
+    if (!answer) {
+      throw new Error("backupFactorKey not found");
+    }
+    
+    let factorKey = await securityQuestion.recoverSecurityQuestionFactor(coreKitInstance, answer);
+    setBackupFactorKey(factorKey);
+    uiConsole("Security Question share: ", factorKey);
+  }
+
+  const logout = async () => {
+    if (!coreKitInstance) {
+      throw new Error("coreKitInstance not found");
+    }
+    await coreKitInstance.logout();
+    setCoreKitStatus(COREKIT_STATUS.NOT_INITIALIZED);
+    uiConsole("Log out");
+    setProvider(undefined);
+  };
+
+  const getUserInfo = (): void => {
+    const user = coreKitInstance?.getUserInfo();
+    uiConsole(user);
+  };
+
+  const exportShare = async (): Promise<void> => {
```

Comment:
this function name should be exportFactor

Reply:
done